storm-core/src/storm.thrift

```diff
@@ -243,6 +243,16 @@ struct SubmitOptions {
   2: optional Credentials creds;
 }
 
+enum NumErrorsChoice {
+  ALL,
+  NONE,
+  ONE
+}
+
+struct GetInfoOptions {
+  1: optional NumErrorsChoice num_err_choice;
```

Comment:
I should add, the current ZK storage of errors isn't very flexible.  We store at most 10 errors per component (hard-coded).  With only 10 errors to work with, I am not sure how much more flexibility we get by doing pagination.  I guess this hard-coded limit of 10 is why I did not pursue pagination—it is just easier to give all 10 errors in the case that we want more than 1.